src/utils/is-feature-included/test/index.js

```diff
@@ -0,0 +1,49 @@
+import test from 'tape';
```

Comment:
Eric would like us to use riteway for new tests. The package is already added, just convert the import and test ;)